src/sql/src/catalog.rs

```diff
@@ -119,9 +119,6 @@ pub trait Catalog: fmt::Debug {
 
     /// Returns the configuration of the catalog.
     fn config(&self) -> &CatalogConfig;
-
-    /// Determines the minimal qualification required to address the named item.
-    fn minimal_qualification(&self, full_name: &FullName) -> PartialName;
```

Comment:
Removed this just in the interest of keeping presently-unused functions out of the API, not because I think it's a fundamentally bad idea to have it available here!